tools/schema_compiler/macos/proto/improbable/entity_state.proto

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015 Improbable Worlds Ltd. All Rights Reserved.
```

Comment:
Can you modify the core sdk download script to clean the mac version of this and `protoc` up?

Reply:
done!